packages/eservice-template-process/src/index.ts

```diff
@@ -1,7 +1,8 @@
 import { genericLogger } from "pagopa-interop-commons";
 import { config } from "./config/config.js";
-import app from "./app.js";
+import { createApp } from "./app.js";
 
+const app = await createApp();
 app.listen(config.port, config.host, () => {
   genericLogger.info(`listening on ${config.host}:${config.port}`);
 });
```

Comment:
we could adapt this to how other processes are booted:
```suggestion
startServer(await createApp(), config);
```

Reply:
We could also take advantage of this PR (or any other API tests PR to be merged soon) to fix this in each process that is still not using startServer

<img width="338" alt="image" src="https://github.com/user-attachments/assets/d9ea987b-e1de-4f0d-b7af-c8ea3b77db6f" />

Reply:
46e6c97d806cc438112e27f139198e122a8c5813
19fea2f259895d0b243eb4d9bf05fc5df05ef0bf